### ISSUE TITLE
AMUSE

### ISSUE THREAD

**Issue by romulogoncalves** [NLeSC/TEAM2018#45]
@ipelupessy it seems there was a suggestion to work on `AMUSE` during the November Sprint, could you provide us some information?

**Comment by ipelupessy** [comment]
yes, I also think this is a good idea, this would be a continuation of the work started in the june sprint
(https://github.com/amusecode/amuse/projects/1). Some issues we couldn't work on then are still open. Furthermore, there is an amuse-framework repository now, used in the ewaterycle project. It basically needs to be cleaned up and and the various variants (AMUSE, OMUSE and HyMUSE) need to take this as a basis.

**Comment by romulogoncalves** [comment]
@ipelupessy could you tell me who was working on it in June?

**Comment by ipelupessy** [comment]
Patrick, Evert, Alessio (1day) and myself

**Comment by romulogoncalves** [comment]
@ipelupessy  is there project hours for this work? If yes, which one?

**Comment by ipelupessy** [comment]
I think last time it was done on eWatercycle hours, but I think the idea was to do it partially on eStep and/ or cloud hours

**Comment by romulogoncalves** [comment]
@ipelupessy were all the goals achieved? Could you summarize in 3 lines what was achieved? Can we consider it done and close this issue?

**Comment by ipelupessy** [comment]
no, we decided to mostly cancel the sprint due to illness (I was not in most of the week), as it would not be very efficient (johan and felipe did not have previous experience with AMUSE)..
